src/Narochno.Slack/SlackClient.cs

```diff
@@ -43,13 +46,68 @@ public SlackClient(HttpClient httpClient, SlackConfig slackConfig)
             message.Channel = message.Channel.Fallback(slackConfig.Channel);
             message.Emoji = message.Emoji.Fallback(slackConfig.Emoji);
 
-            var json = JsonConvert.SerializeObject(message, serializerSettings);
+            string json = JsonConvert.SerializeObject(message, serializerSettings);
 
-            var response = await GetRetryPolicy().ExecuteAsync(() => httpClient.PostAsync(slackConfig.WebHookUrl, new StringContent(json, Encoding.UTF8, "application/json"), ctx));
+            HttpResponseMessage response = await GetRetryPolicy().ExecuteAsync(() => httpClient.PostAsync(slackConfig.WebHookUrl, new StringContent(json, Encoding.UTF8, "application/json"), ctx));

```

Comment:
underscores for fields is fine by me - I don't have ReSharper though for personal use :(